migrations/014.do.top-measurement-participants-mv.sql

```diff
@@ -0,0 +1,36 @@
+CREATE MATERIALIZED VIEW top_measurement_participants_yesterday_view AS
```

Comment:
Do we actually need to keep both materialised views around?

I believe we are already populating the new tables with data. By the time we land this PR next week, we should have all the data required to create the "top measurement participants" view using the new approach.

What happens if this PR simply changes the definition of the existing view?
- No changes in spark-stats are needed; it will return data from the new view.
- The view will contain stats starting this week because we haven't backfilled the underlying tables yet.
- When the backfilling is done, the next refresh will backfill the data in the view too.

I think we can afford to not show historical data for a while, if it means less work & faster delivery of this feature.

Is there anything I missed?

Reply:
Ah great point, a bit embarrassed to have missed it haha. And then we avoid naming issues mentioned above too

Reply:
> And then we avoid naming issues mentioned above too

Yes, exactly!

BTW I run your query for today (removed `- INTERVAL '1 day'`) and got the following data (~7k participants):

https://gist.github.com/bajtos/611dcf3ed5e688a92dc386e7b5f0b6f5

Reply:
Oh, this materialised view shows data for yesterday only, right? So we don't need to worry about missing historical data because that data is not shown in this view anyway.

Reply:
Are you able to verify if it's the same data as the existing view?

Reply:
See https://github.com/filecoin-station/spark-evaluate/pull/341#pullrequestreview-2275461145